packages/manager/apps/dedicated/client/app/account/user/security/user-security.html

```diff
@@ -590,9 +589,8 @@ <h5 data-translate="user_ipRestrictions_list"></h5>
                                             class="center">
                                             <span data-translate="user_ipRestrictions_alert"></span>
                                             <span class="oui-icon oui-icon-help_circle"
-                                                    data-uib-tooltip="{{ 'user_ipRestrictions_alert_information' | translate }}"
+                                                    data-oui-tooltip="{{ 'user_ipRestrictions_alert_information' | translate }}"
                                                     data-tooltip-placement="right"
-                                                    data-tooltip-append-to-body="true"
```

Comment:
No impact ?

Reply:
Append to body will create a `<div>` for tooltip content inside the parent and inherits all parents styles.`<oui-tooltip>` actually creates content inside the parent element by default.
Visually if i remove `append-to-body`, uib tooltip will lose its styles. Since it has been replaced, it looks good